Documentation/metrics.md

```diff
@@ -0,0 +1,26 @@
+Metrics
+=======
+
+cloud-credential-operator (CCO) exposes metrics for integration with Prometheus. Metrics are not persistent. Restarting CCO will reset metrics.
+
+The metrics are exposed via HTTP on port 2112 at `/metrics` (eg. http://localhost:2112/metrics).
+
+The metrics names have a `cco` prefix.
+
+### Server
+
+These metrics describe the status of CCO. 
+
+All these metrics are prefixed with `cco_`
+
+| Name                            | Description                                        | Type      |
+|---------------------------------|----------------------------------------------------|-----------|
+| credentials_requests            | Count of credentials requests                      | Gauge     |
```

Comment:
Would it be ok to put the cco_ prefix on each name just for clarity and copying?